src/Kusto/.gitignore

```diff
@@ -0,0 +1,14 @@
+bin
+obj
+.vs
+generated
+internal
+exports
+custom/*.psm1
+test/*-TestResults.xml
+/*.ps1
+/*.ps1xml
+/*.psm1
+/*.snk
+/*.csproj
+/*.nuspec
```

Comment:
Please ignore `tools` folder as well and don't check in any files in it

Reply:
done

Reply:
Some of the files are already checked in, e.g. `src/Kusto/tools/Resources/examples/readme.md`. Please remove the whole tools/ folder and commit again. Thanks

Reply:
done